raiden/storage/sqlite.py

```diff
@@ -1400,6 +1410,26 @@ def get_light_client_protocol_message_by_identifier(self, identifier):
         )
         return cursor.fetchone()
 
+    def get_light_client_protocol_message_by_internal_identifier(self, internal_msg_identifier: int):
+        cursor = self.conn.cursor()
+        cursor.execute(
+            """
+            SELECT identifier,
+                   message_order,
+                   unsigned_message,
+                   signed_message,
+                   light_client_payment_id,
+                   message_type,
+                   light_client_address,
+                   internal_msg_identifier
```

Comment:
what is the difference between `identifier` and `internal_msg_identifier`?

it seems very suspicious to me that we are using an apparently secondary id in this `WHERE` clause

Reply:
identifier is the message random identifier generated for the payment process, the internal_msg_identifier is the identifier generated by the database, 2 different things

Reply:
i've created issue [RLP-843](https://jirainfuy.atlassian.net/browse/RLP-843) to deal with this.